tests/c_languages/AStyleLintBearTest.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from bears.c_languages.AStyleLintBear import AStyleLintBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+test_file1 = """
+int main()
+{
+    int a = 19;
+    return 0;
+}"""
```

Comment:
please add a line feed before `"""` so the outputted file ends with a line feed.
(Otherwise the linter may create an error about no EOL and EOF.